samples/tutorial/sym1/symDisplay.c

```diff
@@ -0,0 +1,360 @@
+// --------------------------------------------------------------------------
```

Comment:
We don't want any new programs in the tutorial. Please remove it or move it to `targettest/sym1`.

Reply:
Moved to targettest.